includes/wc-term-functions.php

```diff
@@ -151,10 +151,13 @@ function wc_get_product_terms( $product_id, $taxonomy, $args = array() ) {
  * @return int
  */
 function _wc_get_product_terms_name_num_usort_callback( $a, $b ) {
-	if ( $a->name + 0 === $b->name + 0 ) {
+    $aName = (int) $a->name;
```

Comment:
Need to be compatiblity with WordPress coding standards.
So please change `$aName` to `$a_name` and `$bName` to `$b_name`.
Thanks.